test/functionalTest/harnessFunctions.sh

```diff
@@ -715,6 +715,14 @@ function accumulatorStart()
     shift
   fi
 
+  url="/notify"
```

Comment:
This is afix in harness suite? I mean, until this fix the `--url` parameter was being ignored?

Reply:
NTC